### PR TITLE
Fixes DOC-1788

### PR DESCRIPTION
**Pull request by lamagnifica**
@mhoeber @lou-wang @gwprice please take a look.

### CODE REVIEW

File: en_us/video/source/video_uploads.rst [hunk 1]
```diff
@@ -206,6 +206,14 @@ The encoding and hosting process assigns these statuses to video files.
   Processing continues at video hosting sites for 24 hours after you upload a
   file.
 
+* **Imported** files were imported into Studio along with the other assets of
+  a previously exported course. To determine whether processing is complete
+  for a file with a status of **Imported**, refer to the status of the same
+  file on the **Video Uploads page of the original course.
```
Comment (gwprice) [inline en_us/video/source/video_uploads.rst:212]: Missing the trailing *\* on Video Uploads?

**Comment by gwprice** [comment]
The reality is actually a little more complicated. Part of the reason we didn't want to import the status from the source Open edX instance is that the source and destination instances may have completely different video profiles and completely different notions of what is and is not "ready". For example, another instance may use an alternative hosting site instead of Youtube and not consider a video ready until it has been uploaded there. In that case, a video that was ready when exported from edx.org may not be considered ready when imported into another system. Unfortunately, we don't currently have a way to reconcile such discrepancies between deployments, which is why we settled on the "imported" status label. I think the best advice we can give is that any imported video should be tested on a variety of devices and connections to verify that the import is functional.

**Comment by lamagnifica** [comment]
@gwprice apologies, I should have made it clear that the document that I am editing here is the one that is private, shared with our partner organizations only. We do not currently have any outward-facing documentation for the Video Uploads page, and such a document is not currently planned AFAIK. So I am hopeful that the advice given here is useful for our partner teams, using the VAL VEDA pipeline.

**Comment by gwprice** [comment]
Ah. For edX users specifically, I doubt that this would actually come up in the near- to medium-term future, because it would require building a course on and uploading videos to another Open edX instance and then exporting that course and importing it on edx.org.

**Comment by lamagnifica** [comment]
@lou-wang , in light of Greg's expected (lack of) use cases, ok to discard this?

**Comment by lamagnifica** [comment]
per @lou-wang . "i'm ok leaving it out"

**Pull request closed without merging**